test/cmdlineTests/yul_optimizer_steps_without_optimize_blank_string/args

```diff
@@ -0,0 +1,2 @@
+--via-ir --yul-optimizations " "
```

Comment:
@nikola-matic, tried to add a CLI test, but can't find a way to input empty/blank string. In this case, it gets the `"` as the value. I've tried to escape with `\"`, but that doesn't work.
Maybe @r0qs knows how to do this?

Reply:
Not sure I understand - it looks like it's issuing the appropriate error, no? 
```
Error: --yul-optimizations is invalid with a non-empty sequence if Yul optimizer is disabled.
```

Reply:
Yeah, it's not appending the `Note` portion, which is super weird. When running solc with the same params, it works just fine, and the sequence is fully trimmed and thus empty in the debugger. Not really sure what's going on here.

Reply:
Could be a weird bash bug in the cmdline script.

Reply:
Yeah, it's `cmdlineTests.sh` - this is what it runs:
```
solc --via-ir --yul-optimizations '"' '"' yul_optimizer_steps_without_optimize_blank_string/input.sol
```
which then obviously causes the optimization sequence to not be empty empty.

Reply:
It's not a Bash bug. It's that an `args` file is intentionally interpreted by `cmdlineTests.sh` as a simple list of space-separated arguments. It does not support anything fancy, like quoting. Its content is not executed as a Bash command so things like `>`, `|` or variables will not work either. And I think it's better off like this. Support for values with spaces would be nice, but we don't want to bring in all of those other things.

Just make it a Boost test in `test/solc/CommandLineInterface.cpp`.

Reply:
Yeah, CLI boost test will work just as well, although I thought our long term strategy was to move away from boost tests wherever possible.

> It does not support anything fancy, like quoting.

I wouldn't really consider quoting fancy, especially in terms of CLI tests, but I do agree it would be annoying to introduce this into our current cmdline tests.

Reply:
> Yeah, CLI boost test will work just as well, although I thought our long term strategy was to move away from boost tests wherever possible.

Boost tests are not going anywhere any time soon. CLI tests are too crude for some things and we also can't spend time improving them so for the foreseeable future you're free to choose whichever is appropriate.

> I wouldn't really consider quoting fancy, especially in terms of CLI tests, but I do agree it would be annoying to introduce this into our current cmdline tests.

Full Bash-compatible quoting is pretty complex. And something that looks like Bash quoting but acts differently is dangerous because everyone would just assume it works like Bash.